gpio/Makefile

```diff
@@ -32,7 +32,19 @@ endif
 
 #DEBUG	= -g -O0
 DEBUG	= -O2
-CC	= gcc
+
+GCC_ARCH := $(shell gcc -dumpmachine)
+
+ifeq ($(OS),Windows_NT)
+	CROSS_COMPILE ?= arm-linux-gnueabihf-
+else
+	ifneq ($(GCC_ARCH),arm-linux-gnueabihf)
```

Comment:
The indentation is a bit odd here, maybe use spaces with something like:

ifeq ($(OS),Windows_NT)
  CROSS_COMPILE ?= arm-linux-gnueabihf-
else
  ifneq ($(GCC_ARCH),arm-linux-gnueabihf)
    CROSS_COMPILE ?= arm-linux-gnueabihf-
  endif
end

Reply:
converted tabs to spaces